test/workbox-google-analytics/sw/test-initialize.mjs

```diff
@@ -271,35 +283,37 @@ describe(`[workbox-google-analytics] initialize`, function() {
 
     initialize();
 
-    self.dispatchEvent(new FetchEvent('fetch', {
+    await dispatchAndWaitUntilDone(new FetchEvent('fetch', {
       request: new Request(`https://${GOOGLE_ANALYTICS_HOST}` +
           `/collect?${PAYLOAD}`, {
         method: 'GET',
       }),
     }));
 
-    await eventsDoneWaiting();
     clock.tick(100);
 
-    self.dispatchEvent(new FetchEvent('fetch', {
+    await dispatchAndWaitUntilDone(new FetchEvent('fetch', {
       request: new Request(`https://${GOOGLE_ANALYTICS_HOST}/r/collect`, {
         method: 'POST',
         body: PAYLOAD,
       }),
     }));
 
-    await eventsDoneWaiting();
-
     self.fetch.restore();
     sandbox.stub(self, 'fetch').resolves(new Response('', {status: 200}));
 
     clock.tick(100);
 
-    self.dispatchEvent(new SyncEvent('sync', {
-      tag: `workbox-background-sync:workbox-google-analytics`,
-    }));
-
-    await eventsDoneWaiting();
+    // Manually trigger the `onSync` callback in both sync and non-sync
+    // supporting browsers.
+    if ('sync' in registration) {
+      await dispatchAndWaitUntilDone(new SyncEvent('sync', {
+        tag: `workbox-background-sync:workbox-google-analytics`,
+      }));
+    } else {
+      const queue = Queue.prototype.pushRequest.thisValues[0];
```

Comment:
Nit: I found this line to be confusing, and had to trace things back to earlier in the function to figure out that there was a `Spy` set up for `Queue.prototype.pushRequest`, and that `thisValues[0]` was a way of reading info from the `Spy`. 

Would you mind changing this (and the one other usage elsewhere) to something more explicit, like:

```
const pushRequestSpy = sandbox.spy(Queue.prototype, 'pushRequest');
// Later...
const queue = pushRequestSpy.thisValues[0];
```

Reply:
Done.